test/parallel/test-util-deprecate-invalid-code.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const common = require('../common');
+const util = require('util');
+
+[1, true, false, null, {}].forEach(notString => {
+  common.expectsError(() => util.deprecate(() => { }, 'message', notString), {
```

Comment:
Nit: would you mind removing the extra space inside `{ }`? Thanks!

Reply:
Removed. I also changed the commit message. However I see that now the amended forced push to branch isn't picked automatically by the github PR. Any ideas why?

Reply:
Done and I ran linter to ensure things look ok:

make lint-js
Running JS linter...
./node tools/eslint/bin/eslint.js --cache --rulesdir=tools/eslint-rules --ext=.js,.mjs,.md \
	  benchmark doc lib test tools